src/platform_impl/windows/event_loop.rs

```diff
@@ -366,16 +382,26 @@ fn wait_thread(parent_thread_id: DWORD, msg_window_id: HWND) {
             if let Some(wait_until) = wait_until_opt {
                 let now = Instant::now();
                 if now < wait_until {
-                    // MsgWaitForMultipleObjects tends to overshoot just a little bit. We subtract
-                    // 1 millisecond from the requested time and spinlock for the remainder to
-                    // compensate for that.
+                    // Windows' scheduler has a default accuracy of several ms. This isn't good enough for
+                    // `WaitUntil`, so we request the Windows' scheduler to use a higher accuracy if possible.
```

Comment:
```suggestion
                    // `WaitUntil`, so we request the Windows scheduler to use a higher accuracy if possible.
```
nit: This sentence feels weird. Removing the apostrophe makes it feel less so.